docs/changelog.md

```diff
@@ -46,6 +46,9 @@ permalink: /changelog/
 * **concept-engine**:
   * Add new `hasCookieBannerRuleForSession` API in `Engine`. This is currently only supported in the Gecko Engine.
 
+* **pull-to-refresh**:
+  * Enable 'pull to refresh' feature in all channels.
+
```

Comment:
Non-blocking comment: the changelog title is for the component we add a feature towards.

You could omit this entry.

Reply:
We don't have a process around this yet, hence the recommendation to omit it for now.

We probably want to do something like this in the future:

```markdown
* **fenix**
  *  Enable 'pull to refresh' feature in all channels.

* **focus**
  * Enable 'pull to refresh' feature in nightly-only.
```